.github/workflows/first_time_contributor.yml

```diff
@@ -0,0 +1,30 @@
+name: First-time Contributor Welcome
+
+on:
+    issues:
+      types: [opened]
+    pull_request_target:
+      types: [opened, closed]
+
+jobs:
+  first-time-contributor-welcome:
+    runs-on: ubuntu-latest
+    steps:
+    # We don't use the actions/first-interaction action because it can't reference the author, nor can it comment after
+    # PR merge.
+    # Once https://github.com/wow-actions/welcome/pull/12 is merged, we should use https://github.com/wow-actions/welcome
+    # directly.
+    - uses: OrchardCMS/welcome-action@task/update-to-node-20
+      with:
+        FIRST_ISSUE_COMMENT: >
+          Thank you for submitting your first issue, awesome! 🚀 If you haven't filled out the template fully already,
+          please be sure to do so. This helps us better understand your feature request/bug report. A core team member 
+          will be with you shortly to review your issue.
```

Comment:
```suggestion
          Thank you for submitting your first issue! 🚀 We're thrilled to receive your input. If you haven't completed the template yet, please take a moment to do so. This ensures that we fully understand your feature request or bug report. A core team member will review your issue and get back to you.
```

Reply:
Having a very long line makes it a lot harder to work with the text (and auto-wrapping is not really convenient with a whitespace-centric format like YAML). The `>` means that the string will ignore line breaks when processed by the called action, i.e. the actual messages won't have these, it's just for readability in the code file.

Same for https://github.com/OrchardCMS/OrchardCore/pull/15708#discussion_r1563302107.

Reply:
You can re-format the text I suggested so it is wrapped as needed

Reply:
OK, I added these with slight differences.